analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.NullableContext.cs

```diff
@@ -0,0 +1,48 @@
+﻿#nullable enable
+
+using System;
+
+class NullForgivingOperator
+{
+    void Basics(int? i)
+    {
+        _ = i!.Value;               // Compliant, unknown
```

Comment:
It's not. The PR originally contained a single commit, related to Boxing, and it was targeting `Antonio/S3655-cs9-cs10-step6-4` (the previous part of step 6 of the umbrella issue for S3655 implementation for C#).

Merging `Antonio/S3655-cs9-cs10-syntax-step6-4` into `feature/SE` (done by https://github.com/SonarSource/sonar-dotnet/pull/6996) has made GitHub automatically change the target branch of this PR from `Antonio/S3655-cs9-cs10-syntax-step6-4` to `feature/SE`, which means that now the PR contains all commits from `Antonio/S3655-cs9-cs10-syntax-step6-1` to `Antonio/S3655-cs9-cs10-syntax-step6-4` (which were all merged into `feature/SE` yesterday, since they were all blocked by the review of `Antonio/S3655-cs9-cs10-syntax-step6`, taking longer than all the other reviews).

So in conclusion, you have mostly reviewed commits that have already been reviewed in previous PRs, and which have already been merged into `feature/SE`. Sorry for that :-(

Next time I will try to prevent the situation from happening by putting back in draft state a PR with a dependency on another PR, and also signaling to the reviewer that a rebase is required before the PR can be reviewed using the Changes tab. The problem is that we also use draft to signal not to merge, and we allow early reviews of draft PR, so the only way to avoid such a scenario that I currently see is direct communication, issuing a warning.

Reply:
As discussed offline with @martin-strecker-sonarsource, I have rebased onto the new `feature/SE`, and I am going to address the comments on code already merged in a dedicated PR.
This PR is back to the state it was before the auto-change of base branch by GitHub.